akka-http/src/main/scala/akka/http/scaladsl/server/util/JwtSprayJson.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 Lightbend Inc.
+ */
+
+package akka.http.scaladsl.server.util
+
+import pdi.jwt.{ JwtAlgorithm, JwtClaim, JwtHeader, JwtJsonCommon }
+
+import java.time.Clock
+import pdi.jwt.exceptions.JwtNonStringException
+import spray.json._
+
+/**
+ * Implementation of `JwtCore` using `JsObject` from spray-json.
+ *
+ * This class originally came from jwt-spray-json,
+ * but was removed in https://github.com/jwt-scala/jwt-scala/commit/bf1131ce02480103c0b953b97da001105a3ee038
+ */
```

Comment:
We'll need a mention of original author/license/copyright if it was taken from another project (jwt-scala is Apache 2)

Reply:
added now [here](https://github.com/akka/akka-http/pull/4377/commits/0022b48394a6bffec66ef572c81c58664526d44a#diff-7fa80f952d823dc275b3af34fb322b27e4b30d2ab369e80c1fc72cd631e884d5). please check if it's good.